backend/FastAPI.py

```diff
@@ -0,0 +1,47 @@
+from fastapi import FastAPI, UploadFile, File
+import os
+import torchvision
+import torch
+
+app = FastAPI()
+
+
+@app.get("/")
+def hello():
+    return {"API": "API is working fine"}
+
+
+@app.post("/upload_image")
+async def upload_image(filename: str, img_file: UploadFile = File(...)):
+    class_var = 0
+    class_str = "unsucessful"
+    class_str = filename
```

Comment:
Is this necessary? If you want to give a default value you can do this:
```suggestion
    class_str = filename or "unsuccessful"
```

Reply:
Changed code to the suggested solution.